packages/web-components/fast-element/src/attributes.ts

```diff
@@ -56,19 +103,65 @@ export const nullableNumberConverter: ValueConverter = {
     },
 };
 
+/**
+ * An implementation of {@link Accessor} that supports reactivity,
+ * change callbacks, attribute reflection, and type conversion for
+ * custom elements.
+ * @public
+ */
 export class AttributeDefinition implements Accessor {
     private readonly fieldName: string;
     private readonly callbackName: string;
     private readonly hasCallback: boolean;
     private readonly guards: Set<unknown> = new Set();
 
+    /**
+     * The class constructor that owns this attribute.
+     */
+    public readonly Owner: Function;
+
+    /**
+     * The name of the property associated with the attribute.
+     */
+    public readonly name: string;
+
+    /**
+     * The name of the attribute in HTML.
+     */
+    public readonly attribute: string;
+
+    /**
+     * The {@link AttributeMode} that describes the behavior of this attribute.
```

Comment:
Do we have to `@link` all of the non-builtin types? Is this not resolved by TypeScript?

Reply:
This is used by the api-extractor. If you put the `{@link name}` it will turn that into an `<a>` with a link to the docs page for that API.

Reply:
You only need to use it if you want to make a link from within your description or remarks. The types in the API should be linked automatically.